src/time_tracker/invoices/db.clj

```diff
@@ -21,6 +22,23 @@
                         "paid" false})))
 
 (defn retrieve-all
-  "Retrieves a list of all the invoices. No authroization checks yet."
+  "Retrieves a list of all the invoices."
   [connection]
   (retrieve-all-invoices-query {} {:connection connection}))
+
+(defn retrieve
+  "Retrieves a specific invoice."
+  [connection invoice-id]
+  (first (retrieve-invoice-query {:invoice_id invoice-id}
+                                 {:connection connection})))
+
+(defn update!
```

Comment:
@trycatcher can you write a test for this? Even a simple smoke test that just calls the function and verifies that you can mark the invoice as paid would do.